src/frontend/utils/språk.tsx

```diff
@@ -49,6 +50,34 @@ const texts: Record<LocaleType, Record<string, string>> = {
 
 const cache = createIntlCache();
 
+const innholdForLocale = (
+    sanityTekst: LocaleRecordString | LocaleRecordBlock,
+    locale: LocaleType,
+    barnetsNavn?: string
+) => {
+    return typeof sanityTekst[locale] !== 'string' ? (
+        <TekstBlock
+            block={sanityTekst as LocaleRecordBlock}
+            spesifisertLocale={locale}
+            barnetsNavn={barnetsNavn}
+        />
+    ) : (
+        sanityTekst[locale]
+    );
+};
+
+export const sanitizedLocaleRecord = (
+    sanityTekst: LocaleRecordString | LocaleRecordBlock,
+    barnetsNavn?: string
+): Record<LocaleType, string> => {
+    return {
+        //todo: caster til string enn så lenge, til vi får støtte for at Portable Text kan returnere tekst i stedet for ReactNode
```

Comment:
Har sett at det er en pågående tråd om dette på slack, hvor man får støtte for at PortableText kan returnere ren tekst i stedet for en ReactNode